### ISSUE TITLE
Settings interface problem after opkg update (sturgeon)

### ISSUE THREAD

**Issue by tomi7711** [AsteroidOS/asteroid-settings#56]
I updated AsteroidOS with **opkg upgrade** command from ssh. After that I wanted to restart from the Setting but istead of the Setting menu, there is a completely white screen. I can't do anything. I restarted the OS with the button but the problem still exists, I don't see the menu items, just a white screen. Can I do something about it or reflash it?

Thanks!

**Comment by ManuelBoe14** [comment]
I think I have the same issue on lennok.

**Comment by eLtMosen** [comment]
Yes. Sadly, this happens due to OPKG not being suitable for system package updates :/ -> https://github.com/AsteroidOS/asteroid/issues/224
The only known way to recover is to reflash to the latest nightly.

**Comment by tomi7711** [comment]
Ok, no problem :)
It's quick and easy. Works like a charm again!
Thanks! :) I will stay away from opkg after this.

Timo Könnecke ***@***.***> ezt írta (időpont: 2022. okt. 15.,
Szo, 17:07):
